telegram-bot/main.go

```diff
@@ -207,8 +207,19 @@ func main() {
 				}
 			case "nextmeetup":
 				nextmeetup(ID, *client)
+			case "save":
+				check := memberdetails(ID, update.Message.From.ID)
+				if check == true {
+					savenote(ID, update.Message.Text, *client)
+				} else {
+					bot.Send(tbot.NewMessage(ID, "Sorry, only admins can add the details of next meetup."))
+				}
+			case "notes":
+				fetchallnotes(ID, *client)
+			case "deletenote":
+				deletenote(ID, update.Message.Text, *client)
 			default:
-				bot.Send(tbot.NewMessage(ID, "I don't know that command"))
+				fetchnote(ID, update.Message.Command(), *client)
```

Comment:
What does `update.Message.Command()` print?

Reply:
Suppose I send `/hello` to the bot. update.message.command will send `hello` to the bot , so no need to do a regex match later.

Reply:
Umm..but we added the default condition so that it runs when the bot gets an unusual command.